### PR TITLE
a few extra brain damage lines

### PR DESCRIPTION
**Pull request by plapatin**
## About The Pull Request
feat. some of my favorite ahelp messages
i honestly just did this for the sodium chloride line

## Why It's Good For The Game
variety is the spice of life

## Changelog
:cl:
add: Brain-damaged crewmen may now express their frustration about sodium chloride, along with some other phrases
/:cl:

### CODE REVIEW

**Comment by EOBGames** [comment]
The new lines are great, but you've forgotten to include commas at the end of every line.

**Comment by yeeyeh** [comment]
image

**Comment by Iamgoofball** [comment]
Yikes, that's racist

File: strings/traumas.json [hunk 1]
```diff
@@ -118,7 +132,9 @@
         "mr terry",
         "berry",
         "vent hell",
-        "colonel hall"
+        "colonel hall",
+		"manwell",
+		"manual"
```
Comment (Fikou) [inline strings/traumas.json]: shouldnt it be mr manuel

Comment (plapatin) [reply strings/traumas.json]: *shrug

Comment (EOBGames) [reply strings/traumas.json]: Should probably be added just to match up with everything else

**Comment by optimumtact** [comment]
brain damage lines are too ooc as it is.

**Comment by adamkad1** [comment]
orange hate marches on

File: strings/traumas.json [hunk 2]
```diff
@@ -65,7 +65,20 @@
...
+		"@pick(semicolon) WANKER WANKER WANKER WANKER WANKER",
+		"OFFICRE WHEN HE WAS IN SURGERY HIS HEAD FELL OFF @pick(bug)",
+		"i would say srry that was a my bad but i did nothing 2 him thow......",
+		"whats wrong? baby cant escape brig????",
+		"fuckign stop or ill call admin....",
+		"@pick(semicolon)im going to kick you in teh fucking ribs",
+		"he fucking asked if i wa sa ling so i killed him??? @pick(bug)",
+		"I dont have an extraordinary mind",
+		"well iam sorry for the griefes............",
+		"@pick(semicolon)oog is trapped in cage built by oog"
```
Comment (PKPenguin321) [inline strings/traumas.json]: This hit different

**Pull request closed without merging**